services/worker/src/worker/job_runners/config/split_names_from_streaming.py

```diff
@@ -98,7 +89,7 @@ def compute_split_names_from_streaming_response(
     use_auth_token: Union[bool, str, None] = hf_token if hf_token is not None else False
 
     try:
-        split_name_items: List[SplitNameItem] = [
+        split_name_items: List[SplitItem] = [
             {"dataset": dataset, "config": config, "split": str(split)}
```

Comment:
Should we also instantiate here?

```suggestion
            SplitItem({"dataset": dataset, "config": config, "split": str(split)})
```

Reply:
Thanks